ApnsPHP/Message/LiveActivity.php

```diff
@@ -0,0 +1,259 @@
+<?php
+
+/**
+ * This file contains the LiveActivity class.
+ *
+ * SPDX-FileCopyrightText: Copyright 2024 Move Agency Group B.V., Zwolle, The Netherlands
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+namespace ApnsPHP\Message;
+
+use ApnsPHP\Message;
+use AssertionError;
+use UnexpectedValueException;
+
+/**
+ * Live activity style message.
+ */
+class LiveActivity extends Message
+{
+    /**
+     * Content state information
+     * @var array|object
+     */
+    private array|object $state;
+
+    /**
+     * Attributes at the start of an activity
+     * @var array|object
+     */
+    private array|object $attributes;
```

Comment:
Can we defined subtypes, shapes here?

Reply:
No, you're completely free to send whatever you want. All I can do is
```suggestion
    /**
     * Content state information
     * @var array<string,mixed>|object
     */
    private array|object $state;

    /**
     * Attributes at the start of an activity
     * @var array|object
     */
    private array<string,mixed>|object $attributes;
```